demos/wascc/hello-world-assemblyscript/k8s.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hello-world-wascc-assemblyscript
+  annotations:
+    deislabs.io/wascc-action-key: MA6DECXXYRIRPWYJAIKQT7ZBPUCJ7HRBLK3ZSDTGE7WWIJ6VU52PZOCI
```

Comment:
Just a heads up that we are removing the annotation in @bketelsen's upcoming PR in favor of signed modules

Reply:
to be clear, all modules had to be signed already, so we're just removing the extra step and unnecessary annotation

Reply:
okay. Let's cross that bridge when we get there

Reply:
oh wait we merged that PR. So the recommendation is to remove this annotation now?

Reply:
Yes, the annotation can go 😄